Description: Do not install stuff to /opt
Author: Jonas Genannt <jonas.genannt@capi2name.de>
Forwarded: not-needed

Index: carbon/setup.cfg
===================================================================
--- carbon.orig/setup.cfg
+++ /dev/null
@@ -1,9 +0,0 @@
-[install]
-prefix = /opt/graphite
-install-lib = %(prefix)s/lib
-
-[bdist_rpm]
-requires = python-twisted
-           whisper
-
-post-install = distro/redhat/misc/postinstall
